Intermediate/MultiReplaceInMeasures.csx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Title: Multi-string Search and Replace in any selected measure
+ * 
+ * Author: Dan Meissner
+ * 
+ * This script will loop through a matched pair of strings held in a 2-D array 
+ * and search and replace a FromString and ToString pair in the currently selected measures.

```

Comment:
How would you feel about updating this section to read like this?
> This script iterates through all currently-selected measures, using the matched pairs of strings 
> in the 2-D array named "ReplacementPair" below as a template, replacing all instances of "FromStringX" 
> in the current measure's DAX expression with the corresponding "ToStringX"

Reply:
Much better said :)  Looking through some of the comments I put in there months ago, it must have been late at night.  Thanks for looking at this with a second, objective set of eyes.